modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/api/SessionImpl.java

```diff
@@ -333,9 +334,9 @@ public static <T> T await(CompletionStage<T> stage) {
         try {
             return stage.toCompletableFuture().get();
         } catch (ExecutionException e) {
-            throw new IgniteException(e.getCause());
+            throw new IgniteException(OPERATION_INTERRUPTED_ERR, e.getCause());
```

Comment:
I think we should use SqlException here and below

Reply:
Done, thanks